### ISSUE TITLE
`sluggable:table` command not working in L5

### ISSUE THREAD

**Issue by kilburn** [cviebrock/eloquent-sluggable#105]
The `sluggable:table` command fails when using the current `master` version in **Laravel 5.0.2**.

As shown in the output below, the problem is a wrong path to the `migrations` folder, which has been moved from `app/database/migrations` in *Laravel 4* to `database/migrations` in *Laravel 5*.

```bash
$ artisan sluggable:table tour
exception 'ErrorException' with message 'file_put_contents(/srv/web/app/database/migrations/2015_02_14_122923_add_sluggable_columns.php): failed to open stream: No such file or directory' in /srv/web/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php:74
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'file_put_conten...', '/srv/web/vendor...', 74, Array)
#1 /srv/web/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(74): file_put_contents('/srv/web/app/da...', '<?php??use Illu...', 0)
#2 /srv/web/vendor/laravel/framework/src/Illuminate/Database/Migrations/MigrationCreator.php(51): Illuminate\Filesystem\Filesystem->put('/srv/web/app/da...', '<?php??use Illu...')
#3 /srv/web/vendor/cviebrock/eloquent-sluggable/src/SluggableTableCommand.php(49): Illuminate\Database\Migrations\MigrationCreator->create('add_sluggable_c...', '/srv/web/app/da...')
#4 /srv/web/vendor/cviebrock/eloquent-sluggable/src/SluggableTableCommand.php(29): Cviebrock\EloquentSluggable\SluggableTableCommand->createBaseMigration()
#5 [internal function]: Cviebrock\EloquentSluggable\SluggableTableCommand->fire()
#6 /srv/web/vendor/laravel/framework/src/Illuminate/Container/Container.php(523): call_user_func_array(Array, Array)
#7 /srv/web/vendor/laravel/framework/src/Illuminate/Console/Command.php(115): Illuminate\Container\Container->call(Array)
#8 /srv/web/vendor/symfony/console/Symfony/Component/Console/Command/Command.php(253): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 /srv/web/vendor/laravel/framework/src/Illuminate/Console/Command.php(101): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 /srv/web/vendor/symfony/console/Symfony/Component/Console/Application.php(874): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 /srv/web/vendor/symfony/console/Symfony/Component/Console/Application.php(195): Symfony\Component\Console\Application->doRunCommand(Object(Cviebrock\EloquentSluggable\SluggableTableCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 /srv/web/vendor/symfony/console/Symfony/Component/Console/Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#13 /srv/web/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(91): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#14 /srv/web/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#15 {main}
```

**Comment by kilburn** [comment]
I have fixed this issue along with some others (such as a wrong path to the `stubs` folder) in #104

**Comment by kilburn** [comment]
Closed in 7de0c2e
